test/test_torch.py

```diff
@@ -10420,33 +10420,38 @@ def run_test(tensor_dims, some):
     @unittest.skipIf(not TEST_NUMPY, "Numpy not found")
     def test_quantile(self, device, dtype):
         # Generate some random test cases
+        ops = ['quantile', 'nanquantile']
```

Comment:
I like how you're modifying this test to iterate over both ops. What about new special cases where:

- all values being reduced are nan
- all values but one being reduced are nan
- all values being reduced in one row are nan but values in another row are not exclusively nan

Would this cover the new behavior?